atomic-structure/content.md

```diff
@@ -0,0 +1,25 @@
+# Radioactive Decay
+#### Learning objective
+Basic understanding of atomic structure
```

Comment:
I would prefer imperative. I lean to simple and direct if a wide audience is going to be using and modifying the nodes.